test/verify/check-system-terminal

```diff
@@ -120,6 +120,10 @@ PROMPT_COMMAND='printf "\033]0;%s@%s:%s\007" "${USER}" "${HOSTNAME%%.*}" "${PWD/
         wait_line(n, prompt)
         b.wait_not_in_text(".terminal .xterm-accessibility-tree", "disconnected")
 
+        # Check that closing the terminal tab is guarded by a confirmation dialog
+        b.close()
+        b.handle_alert(accept=False)
```

Comment:
This [often fails](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19148-20230822-140007-3bf44846-debian-stable-other/log.html#190-1) with "No dialog is showing". I. e. this needs some waiting loop for a dialog to actually appear. That isn't part of the DOM, and there is no JS API for checking whether an alert or confirm dialog is present (as both functions are blocking). But you could change `handle_alert` to retry a few times if it returns that precise error message?